frontend/src/components/pages/CampOverview/ManageSessions/ManageSessionsModalTable.tsx

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TableContainer,
+  Input,
+  Tag,
+  HStack,
+  Text,
+  Button,
+  FormControl,
+} from "@chakra-ui/react";
+import { ManageCampSessionDetails } from "../../../../types/CampsTypes";
+import { getFormattedSessionDatetime } from "../../../../utils/CampUtils";
+
+interface SessionRowTitleProps {
+  session: ManageCampSessionDetails;
+  index: number;
+}
+
+const SessionRowTitle = ({
+  session,
+  index,
+}: SessionRowTitleProps): JSX.Element => {
+  return (
+    <HStack spacing={1}>
+      <Text textStyle="buttonSemiBold" color="text.default.100">
+        Session {index + 1}
+      </Text>
+      <Text
+        textStyle="buttonSemiBold"
+        color={
+          session.registeredCampers === session.capacity
+            ? "text.critical.100"
+            : "text.default.100"
+        }
+      >
+        ({session.registeredCampers}/{session.capacity})
+      </Text>
+    </HStack>
+  );
+};
+
+interface ManageSessionsModalTableProps {
+  campStartTime: string;
+  campEndTime: string;
+  sessions: Array<ManageCampSessionDetails>;
+  deletedSessions: Set<string>;
+  updatedCapacities: Map<string, number | undefined>;
+  onEditCapacity: (
+    updatedSessionId: ManageCampSessionDetails,
+    updatedCapacity: string,
+  ) => void;
+  onClickDeleteSession: (deleteSessionId: ManageCampSessionDetails) => void;
+}
+
+const ManageSessionsModalTable = ({
+  campStartTime,
+  campEndTime,
+  sessions,
+  deletedSessions,
+  updatedCapacities,
+  onEditCapacity,
+  onClickDeleteSession,
+}: ManageSessionsModalTableProps): JSX.Element => {
+  const sessionIsFull = (session: ManageCampSessionDetails): boolean =>
+    session.capacity === session.registeredCampers;
+
+  return (
+    <TableContainer>
+      <Table variant="simple">
+        <Thead>
+          <Tr>
+            <Th textTransform="none">
+              <Text textStyle="bodyRegular">Camp Session</Text>
```

Comment:
Unless we decided to change these sizes, bodyRegular is supposed to be 18px I think. Is this something we need to change in textStyles file?
<img width="1071" alt="image" src="https://user-images.githubusercontent.com/57159970/193187842-dd78be32-f964-4b85-a097-5ed9b3588710.png">